app/views/groups/_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="container">
+  <%= simple_form_for group do |f| %>
+    <%= f.input :name %>
+    <%= f.input :responsable_name %>
+    <%= f.input :responsable_contact %>
+    <%= f.input :from_date, order: [:day, :month, :year] %>
+    <%= f.input :to_date, order: [:day, :month, :year] %>
+    <%= f.input :notes %>
+    <%= f.submit t(".submit") %>
```

Comment:
A question: do we need to explicitly set the text for the button here? or can simple form pick it up magically from the translation file?

Reply:
I'm sorry @elle but I didn't understand. Do you mean to use just: `<%=  f.submit %>`  without `t(".submit")`?

Reply:
Yes, exactly

Reply:
Done in 1323adf.